manager/manager/types.go

```diff
@@ -54,21 +55,19 @@ type Repository interface {
 // Deployment defines a deployment that describes
 // the creation, modification and/or deletion of a set of resources.
 type Deployment struct {
-	Name       string               `json:"name"`
-	ID         int                  `json:"id"`
-	CreatedAt  time.Time            `json:"createdAt,omitempty"`
-	DeployedAt time.Time            `json:"deployedAt,omitempty"`
-	ModifiedAt time.Time            `json:"modifiedAt,omitempty"`
-	DeletedAt  time.Time            `json:"deletedAt,omitempty"`
-	Status     DeploymentStatus     `json:"status,omitempty"`
-	Current    *Configuration       `json:"current,omitEmpty"`
-	Manifests  map[string]*Manifest `json:"manifests,omitempty"`
```

Comment:
It seems like we're losing what the last manifest was that was deployed on the deployment. We have the current config, but it would be good to at least store the name of the manifest that led to the status.